core/profiles/standard/config/layout.layout.home.json

```diff
@@ -114,6 +117,47 @@
                 }
             }
         },
+        "fe8c7715-c7e9-46d0-8b09-09afb78b2891": {
+            "plugin": "views:card_grid-block",
+            "data": {
+                "module": "views",
+                "delta": "card_grid-block",
+                "settings": {
+                    "link_to_view": 0,
+                    "more_link": false,
+                    "use_pager": false,
+                    "pager_id": 0,
+                    "items_per_page": "3",
+                    "offset": 0,
+                    "path": "card-grid",
+                    "fields_override": null,
+                    "title_display": "none",
+                    "title": "",
+                    "style": "dynamic",
+                    "block_settings": [],
+                    "contexts": [],
+                    "allowed": [],
+                    "path_override": false,
+                    "exposed": [],
+                    "admin_label": "",
+                    "admin_description": ""
+                },
+                "uuid": "fe8c7715-c7e9-46d0-8b09-09afb78b2891",
+                "style": {
+                    "plugin": "dynamic",
+                    "data": {
+                        "settings": {
+                            "classes": "",
+                            "wrapper_tag": "div",
+                            "title_tag": "h2",
+                            "title_classes": "block-title",
+                            "content_tag": "div",
+                            "content_classes": "container card-design"
```

Comment:
Out of curiosity: why the "container" class?

Reply:
I think the reason is that the `l-top` region hasn't a container class on a inner wrapper, at least in the Home layout.